hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/webapp/util/WebAppUtils.java

```diff
@@ -509,11 +512,18 @@ static String getPassword(Configuration conf, String alias) {
       char[] passchars = conf.getPassword(alias);
       if (passchars != null) {
         password = new String(passchars);
+        LOG.info("Successful password retrieval for alias: {}", alias);
       }
     }
     catch (IOException ioe) {
       password = null;
+      LOG.error("Unable to retrieve password for alias: {}", alias, ioe);
     }
+
+    if (password == null) {
+      LOG.error("Password does not exist for alias: {}", alias);
```

Comment:
If the password is null, that means execution will fail, or there can be scenarios where this can be ignored? If it can be i think maybe WARN level is enough here

Reply:
I think it could happen in some cases when the password does not exist for the following test.

`// let's make sure that a password that doesn't exist returns null
  assertNull(WebAppUtils.getPassword(conf, "invalid-alias"));`
  
I will change it to WARN level :)